notebooks/sprints/sprint19/530_rsclient_staging.ipynb

```diff
@@ -0,0 +1,520 @@
+     "output_type": "stream",
+     "text": [
+      "Session S1A_20231120061537234567 has 60 assets with datetime 2023-11-20T06:15:37.234000+00:00\n",
+      "Session S1A_20220715090550123456 has 40 assets with datetime 2022-07-15T09:05:50.123000+00:00\n",
+      "Session S1A_20210410031928012345 has 21 assets with datetime 2021-04-10T03:39:28.012000+00:00\n",
+      "Session S1A_20200105072204051312 has 60 assets with datetime 2020-01-05T18:52:26.165000+00:00\n"
+     ]
+    }
+   ],
+   "source": [
+    "result = session.get(f\"{href}/cadip/collections/{cadip_collection_id}/items\")\n",
+    "items_collection = result.json()\n",
+    "assert items_collection.get(\"type\") == \"FeatureCollection\"\n",
+    "assert len(items_collection.get(\"features\")) > 0\n",
+    "for item in items_collection.get(\"features\"):\n",
+    "    print(f\"Session {item.get('id')} has {len(item.get('assets'))} assets with datetime {item.get('properties').get('datetime')}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9915607a",
+   "metadata": {},
+   "source": [
+    "### Getting all the assets from the aux_interval collection found in the configuration of ADGS station\n",
+    "\n",
+    "We will deliberately alter the value of a download link for one of the files, causing the staging process to fail at a certain point."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "de1183f3-3861-4838-a6ce-9debf346a666",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "AUXIP asset S1A_OPER_AUX_RESORB_OPOD_20231218T110702_V20231218T071044_20231218T102814.EOF has datetime 2023-12-18T00:00:00Z\n",
+      "AUXIP asset S1A_OPER_AUX_RESORB_OPOD_20230405T110702_V20230405T071044_20230405T102814.EOF has datetime 2023-04-05T00:00:00Z\n",
+      "AUXIP asset S2__OPER_AUX_ECMWFD_PDMC_20230216T120000_V20190217T090000_20190217T210000.TGZ has datetime 2023-02-17T09:00:00Z\n",
+      "AUXIP asset S1A_OPER_AUX_PREORB_OPOD_20230122T062732_V20230122T062732_20230122T062732.EOF has datetime 2023-01-22T00:00:00Z\n",
+      "AUXIP asset S1A_OPER_MPL_ORBPRE_20210214T021411_20210221T021411_0001.EOF has datetime 2021-02-21T00:00:00Z\n",
+      "AUXIP asset S1A_OPER_AUX_OBMEMC_PDMC_20210123T000000.xml has datetime 2021-01-23T00:00:00Z\n",
+      "AUXIP asset S1A_OPER_AUX_PREORB_OPOD_20201002T062732_V20201002T053140_20201002T120640.EOF has datetime 2020-10-02T00:00:00Z\n",
+      "AUXIP asset S1A_OPER_MPL_ORBSCT_20200829T150704_99999999T999999_0025.EOF has datetime 2020-08-29T00:00:00Z\n",
+      "AUXIP asset S2__OPER_AUX_ECMWFD_PDMC_20200216T120000_V20190217T090000_20190217T210000.TGZ has datetime 2020-02-17T09:00:00Z\n",
+      "AUXIP asset S2__OPER_AUX_ECMWFD_PDMC_20190216T120000_V20190217T090000_20190217T210000.TGZ has datetime 2019-02-17T09:00:00Z\n"
+     ]
+    }
+   ],
+   "source": [
+    "result = session.get(f\"{href_adgs}/auxip/collections/{adgs_collection_id}/items\")\n",
```

Comment:
@vprivat-ads same here, AuxipClient should inherit pystac_client.Client so we can use its wrapped functions.